src/main/lib/db.ts

```diff
@@ -1,8 +1,5 @@
 import pg from 'pg'
```

Comment:
I don't like that you're moving this specific logic out of this file and into index.ts - in general there are a lot of arbitrary refactors in this PR that I don't agree with, rather than simply focusing on implementing the main change